src/status_im/ui/screens/chat/components/input.cljs

```diff
@@ -607,19 +635,21 @@
                                :refs                   refs
                                :set-active-panel       #()}]]]]
                ;CONTROLS
-               [rn/view {:style (styles/new-bottom-sheet-controls insets)}
-                [quo2.button/button {:icon true :type :outline :size 32} :main-icons2/image]
-                [rn/view {:width 12}]
-                [quo2.button/button {:icon true :type :outline :size 32} :main-icons2/reaction]
-                [rn/view {:flex 1}]
-                ;;SEND button
-                [rn/view {:ref send-ref :style (when-not (seq (get @input-texts chat-id)) {:width 0 :right -100})}
-                 [quo2.button/button {:icon     true :size 32 :accessibility-label :send-message-button
-                                      :on-press #(do (swap! context assoc :clear true)
-                                                     (clear-input chat-id refs)
-                                                     (re-frame/dispatch [:chat.ui/send-current-message]))}
-                  :main-icons2/arrow-up]]]
+               (when-not (seq suggestions)
+                 [rn/view {:style (styles/new-bottom-sheet-controls insets)}
+                  [quo2.button/button {:icon true :type :outline :size 32} :main-icons2/image]
+                  [rn/view {:width 12}]
+                  [quo2.button/button {:icon true :type :outline :size 32} :main-icons2/reaction]
+                  [rn/view {:flex 1}]
+                  ;;SEND button
+                  [rn/view {:ref send-ref :style (when-not (seq (get @input-texts chat-id)) {:width 0 :right -100})}
+                   [quo2.button/button {:icon     true :size 32 :accessibility-label :send-message-button
+                                        :on-press #(do (swap! context assoc :clear true)
+                                                       (clear-input chat-id refs)
+                                                       (re-frame/dispatch [:chat.ui/send-current-message]))}
```

Comment:
I believe we have agreed to use `>evt` for re/frame dispatches so can you update this here

Reply:
I was wondering wouldn't it better to call the alias >dispatch since it "dispatches an event", like <sub alias "subscribes to an event"

Reply:
> I was wondering wouldn't it better to call the alias >dispatch since it "dispatches an event", like <sub alias "subscribes to an event"

cc @flexsurfer